x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -566,82 +578,87 @@ private void indexBwcInvalidation(UserToken userToken, ActionListener<Boolean> l
     private void indexInvalidation(String tokenDocId, Version version, ActionListener<Boolean> listener, AtomicInteger attemptCount,
                                    String srcPrefix, long documentVersion) {
         if (attemptCount.get() > 5) {
+            logger.warn("Failed to invalidate token [{}] after [{}] attempts", tokenDocId, attemptCount.get());
```

Comment:
Set a `MAX_ATTEMPT_COUNT` (since it's used in 4 places already) to 5 and also use that then in the log message instead of `attemptCount.get()` ?

Reply:
I've extracted the constant, but I prefer to leave the `get` as it reflects that actual number of attempts, not the theoretical max (which _should_ be the same, but I prefer to have the debug reflect reality rather than theory in case they diverge).